specs/Device/Device/schema.json

```diff
@@ -113,6 +113,11 @@
           "minimum": 0,
           "maximum": 1
         },
+        "rssi": {
+          "type": "number",
+          "minimum": -1,
```

Comment:
could you devise a more sophisticated schema

```
"oneOf": [{
  "const": -1,
},
{
    "maximum": 1,
   "minimum": 0
}
]
```

Reply:
@jmcanterafonseca Sorry. I am new to JSON Schema. It didn't occur to me that I can mix same `type` in `oneOf`.